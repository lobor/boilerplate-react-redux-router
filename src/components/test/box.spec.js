import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Box from './../box'

test('render right box', () => {
  const tree = renderer.create(<Box />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render left box', () => {
  const tree = renderer.create(<Box left />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render center box', () => {
  const tree = renderer.create(<Box center />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render space around box', () => {
  const tree = renderer.create(<Box spaceAround />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render space-between box', () => {
  const tree = renderer.create(<Box spaceBetween />).toJSON()
  expect(tree).toMatchSnapshot()
})
