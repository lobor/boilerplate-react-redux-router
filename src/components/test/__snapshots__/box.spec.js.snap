// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render center box 1`] = `
.c0 {
  font-size: 1rem;
  padding: 5px 10px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 4%;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0:last-child {
  margin-right: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-right: 0;
  }
}

<div
  className="c0"
/>
`;

exports[`render left box 1`] = `
.c0 {
  font-size: 1rem;
  padding: 5px 10px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 4%;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

.c0:last-child {
  margin-right: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-right: 0;
  }
}

<div
  className="c0"
/>
`;

exports[`render right box 1`] = `
.c0 {
  font-size: 1rem;
  padding: 5px 10px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 4%;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0:last-child {
  margin-right: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-right: 0;
  }
}

<div
  className="c0"
/>
`;

exports[`render space around box 1`] = `
.c0 {
  font-size: 1rem;
  padding: 5px 10px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 4%;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c0:last-child {
  margin-right: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-right: 0;
  }
}

<div
  className="c0"
/>
`;

exports[`render space-between box 1`] = `
.c0 {
  font-size: 1rem;
  padding: 5px 10px;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 4%;
  -webkit-letter-spacing: .1rem;
  -moz-letter-spacing: .1rem;
  -ms-letter-spacing: .1rem;
  letter-spacing: .1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-between;
  -webkit-justify-content: space-between;
  -ms-flex-pack: space-between;
  justify-content: space-between;
}

.c0:last-child {
  margin-right: 0;
}

@media (max-width:25em) {
  .c0 {
    margin-right: 0;
  }
}

<div
  className="c0"
/>
`;
